examples/Backtesting/full_lookup.py

```diff
@@ -93,10 +93,10 @@
 ### Constructing campaigns for the simulation loop
 
 # In this example, we create several campaigns.
-# First let us create three campaigns that each use a different chemical encoding to
+# First let us create two campaigns that each use a different chemical encoding to
```

Comment:
still three, no?

Reply:
yes, I added back later and overlooked this